lib/ansible/modules/network/ios/ios_static_route.py

```diff
@@ -236,7 +251,7 @@ def main():
     want = map_params_to_obj(module, required_together=required_together)
```

Comment:
Not required to pass `required_together ` separately. It is stored as module attribute and value can be retrieved using `module.required_together`

Reply:
For the same reason we can't use it directly, `required_together` doesn't get added to the module, so we need to pass it along. We _could_ instead define it in `params_to_obj()`, but it seems less surprising to define it with the others.